applications/Web3-deliberative-governance-stack.md

```diff
@@ -0,0 +1,213 @@
+
+#### Daniel Ospina, RnDAO 
+
+Daniel Ospina is a specialist in decentralized governance and DAO operations, co-founder of RnDAO, and recently has served as Head of Governance at Aragon and led the lunch of the Aragon Network DAO. Previously, Daniel worked as an organization design facilitator for web2 and web3 organizations, was a visiting lecturer at Oxford University, started two companies, turned around a family enterprise, co-founded an innovation community, and was a chef in an experimental kitchen. He advised and facilitated for clients like Google, BCG, Daimler, and the UNDP. You can follow his work on platforms like TEDx, Harvard Business Review, and Mirror.
+
+### Team Code Repos
+
+Not relevant at this stage
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/antoinevergne/
+- https://www.linkedin.com/in/remybourganel/
+- https://www.linkedin.com/in/oliviermauco/
+
+## Development Status :open_book:
+
+### Publications 
+
+The first presentation of the problem and path to solution: https://archive.org/details/vergne-metagov
+
+Bourganel, Remy, 2016, Algorithms, diachronic forms, and metadesign, https://www.researchgate.net/publication/305728211_Algorithms_diachronic_forms_and_metadesign 
+
+### references to conversations you might have had related to this project with anyone from the Web3 Foundation,
+
+Discussions with the grant team: David Hawig and Seraya Takahashi, mail conversation with Sebastian Müller
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+- **Total Estimated Duration:** 6 months for milestones 1 and 6 months for milestones 2 / 1 year for milestones 3 and 4 and one year for milestones 5 
+- **Full-Time Equivalent (FTE):**  1,5 FTE for milestones 1 and 2 / 2 FTE for milesteones 3 and 4 and 2 FTE for milestone 5
+- **Total Costs:** The full process of all Milestones demands extended funding. But we do not seek complete funding now. Each milestone can be funded in a consequent order, or partially in combination with other grant-making programs. **The current grant application aims at funding Milestone 1** and if successful milestone 2 which will allow unlocking Milestones 3 to 5. Funding for milestones 3 to 5 is difficult to estimate at this stage. Milestone 1: $25.000 / Milestone 2: $58.000. 
```

Comment:
The text strongly suggests that you are seeking funding for M1, but it's still a bit unclear here. It'd be easiest if you just put one number here, and--if you are not asking for funding for M2--move that milestone to `Future Work`. 

If you are looking for guidance here: it would be a lot easier to only fund M1 now (below 30k, i.e. 3 approvals). That leaves M2 in the air (and M2 by itself is a Level 3 grant), but follow-up grants are usually easier to approve if the committee found the previous work well done.